src/storage/src/server.rs

```diff
@@ -34,6 +35,8 @@ pub struct Config {
     pub now: NowFn,
     /// Configuration for source and sink connection.
     pub connection_context: ConnectionContext,
+    /// Other configuration for sources and sinks.
```

Comment:
This is "instance" as in "storage instance" right?

```suggestion
    /// Other configuration for storage instances.
```